### ISSUE TITLE
About 'cube_inputs' for unifuse.py.

### ISSUE THREAD

**Issue by LZleejean** [alibaba/UniFuse-Unidirectional-Fusion#6]
[Question]
`cube_inputs = torch.cat(torch.split(input_cube_image, self.cube_h, dim=-1), dim=0)` in `unifuse.py`.
this code seemingly cat six cube face on dim0. i.e., (batch_size, 3, h, 6*h) --> (batch_size*6, 3, h, h).
I think that the cube encoder can not ultize feature of different minibatch. Intuitively, catting these face on dim of channel are responable. But, it can't use the imagenet-pretrained network.
Did I understand correctly?

**Comment by HalleyJiang** [comment]
This line of code definitely casts input_cube_image from (batch_size, 3, h, 6h)  to (batch_size*6, 3, h, h). 
In Unifuse, the cube decoder just processes each cube face independently. Maybe to cat the cube faces in the channel dim may help to jointly model the cubemap better. The imagenet-pretrained parameters can be also utilized. We can copy the parameters of the first layer several times to initialize the new defined layer. Please refer to https://github.com/nianticlabs/monodepth2/blob/93b9ebb4c8bdc71a07e18bee64d76ec21bc3e8fc/networks/resnet_encoder.py#L42.

**Comment by LZleejean** [comment]
> This line of code definitely casts input_cube_image from (batch_size, 3, h, 6h) to (batch_size*6, 3, h, h).
> In Unifuse, the cube decoder just processes each cube face independently. Maybe to cat the cube faces in the channel dim may help to jointly model the cubemap better. The imagenet-pretrained parameters can be also utilized. We can copy the parameters of the first layer several times to initialize the new defined layer. Please refer to https://github.com/nianticlabs/monodepth2/blob/93b9ebb4c8bdc71a07e18bee64d76ec21bc3e8fc/networks/resnet_encoder.py#L42.

Thank you for your suggestion.
